index.html

```diff
@@ -1569,14 +1575,14 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
 
   <section><h3>Releasing NFC</h3>
   <p>
-    To <dfn>release NFC</dfn> on a <a>settings object</a>, perform the following steps:
+    To <dfn>release NFC</dfn> on an <a>environment settings object</a>, perform the following steps:
   </p>
   <ol id="steps-nfc-release">
     <li>
       <a>Suspend NFC</a>.
     </li>
     <li>
-      For each pending <a>NFCWriter.push</a> associated with the <a>settings object</a>:
+      For each pending <a>NFCWriter.push</a> associated with the <a>current settings object</a>:
```

Comment:
This uses current but the algorithm takes a different ESO as an argument. Is that intentional?

Reply:
I don't think "current" makes much sense here; probably better to use the one associated with the document. Pushed a patch to clarify.